### PR TITLE
List of former (and current) RST developers

### PR DESCRIPTION
**Pull request by ecbland**
As part of our discussion about RST authorship criteria, we agreed to create a list of former RST developers to distribute alongside the software. This PR adds that list to the upcoming release. 

I have also included the current developers, since I think this will make the list easier to maintain.

It's almost guaranteed that I have missed some people, and mischaracterized some people's contributions...please correct any errors that you notice! For some people, I deduced their contributions by searching for author names in the source code, so I didn't have much to go by :)

### CODE REVIEW

**Comment by mtwalach** [comment]
Looks good as far as I can tell - might be worth (informally if possible to speed things up) to check with PIs as they may have some extra historical knowledge.

File: former-developers.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
+- A.G. Burrell, documentation, bugfixes, testing
+- R.A. Greenwald, `fitacfex2`
+- K. Kotyk, `fitacf3.0`
+- P.V. Ponomarenko, DAWG Chair (2013-2019), `fitacf2.X`, `fitacf3.0`, `make_sim`
+- A.S. Reimer, refactorization, documentation, bugfixes
+- A.J. Ribeiro, `make_sim`
```
Comment (egthomas) [inline former-developers.md:14]:
```suggestion
- A.J. Ribeiro, `fitex2`, `lmfit`, `make_sim`
- J.M. Ruohoniemi, Map Potential, gridding and geolocation libraries
```

File: former-developers.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
+- A.G. Burrell, documentation, bugfixes, testing
```
Comment (egthomas) [inline former-developers.md:9]:
```suggestion
- A.G. Burrell, documentation, bugfixes, testing
- E.D.P. Cousins, CS10 and PSR10 convection models
```

File: former-developers.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
```
Comment (egthomas) [inline former-developers.md:8]: Bill might be able to clarify but I believe Mike wrote all of the original gridding software (in IDL); maybe Bill or Rob ported it to C.

File: former-developers.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
```
Comment (egthomas) [inline former-developers.md:4]: While I can't speak to the full development history, if anyone should receive primary credit for the original FITACF algorithm it should probably be Kile.

File: former-developers.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
+- A.G. Burrell, documentation, bugfixes, testing
+- R.A. Greenwald, `fitacfex2`
```
Comment (egthomas) [inline former-developers.md:10]:
```suggestion
- R.A. Greenwald, `fitacfex`
```

While Ray's contributions are probably immeasurable, he and Kjellmar developed the original `fitacfex` algorithm while AJ developed `fitacfex2` (with Ray's input I imagine)

File: former-developers.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+- K. Kotyk, `fitacf3.0`
+- P.V. Ponomarenko, DAWG Chair (2013-2019), `fitacf2.X`, `fitacf3.0`, `make_sim`
+- A.S. Reimer, refactorization, documentation, bugfixes
+- A.J. Ribeiro, `make_sim`
+- M.T. Schmidt, refactorization, documentation
+- S.G. Shepherd, `AACGM_v2`, `MLT_v2`, `elevation_v2`, TS18 convection model
```
Comment (egthomas) [inline former-developers.md:16]:
```suggestion
- S.G. Shepherd, `AACGM_v2`, `MLT_v2`, `IGRF_v2`, `elevation_v2`, TS18 convection model
```

**Review by egthomas — COMMENTED** [review]
A few brief suggestions, no doubt far from comprehensive.

File: former-developers.md (hunk 4 above)
Comment (ecbland) [reply former-developers.md:4]:
```suggestion
- K. Baker, original JHUAPL author, `fitacf1.0`
```

File: former-developers.md (hunk 3 above)
Comment (ecbland) [reply former-developers.md:8]: Just checked again...Bill's name appears only in one place in the source code: `gtable.c` I'll clarify with him and others listed here
```suggestion
- W.A. Bristow, GridTableMap
```

**Comment by ecbland** [comment]
Thanks @egthomas! I will run this by a few PIs to try to dig up some more info :)

File: former-developers.md (hunk 3 above)
Comment (ecbland) [reply former-developers.md:8]: Bill has indicated that he should be removed from this list, so I will just commit this change now...

```suggestion
```

**Comment by ecbland** [comment]
Quick update: a few PIs have offered to help track down some more detailed information about the original RST authors. I don't know how long this will take, so let's push ahead with the v4.5 release and we can merge this into `develop` at a later time.

**Comment by ecbland** [comment]
My most recent commit here includes some info I got from Ray and Kile. I wasn't able to get much specific information from them about who did what, so I just listed the names.

**Comment by mts299** [comment]
Where should we host this? Sorry, maybe this was mentioned in our meeting. I can add it to readthedocs if you want.

File: rst-authors.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- E.D.P. Cousins, CS10 and PSR10 convection models
+- K. Kotyk, FITACF_v3
+- P.V. Ponomarenko, DAWG Chair (2013-2019), FITACF_v2, FITACF_v3, data simulator library
+- A.S. Reimer, refactorization, documentation, bugfixes
+- A.J. Ribeiro, FITEX2, LMFIT, data simulator library
+- M.T. Schmidt, refactorization, documentation
+- S.G. Shepherd, AACGM_v2, MLT_v2, IGRF_v2, elevation_v2, TS18 convection model
+- K.T. Sterne, DAWG Co-Chair (2019-), code comments
+- E.G. Thomas, TS18 convection model, IDL gridding library, sounding library, refactorization, modularization, code comments
+- M.-T. Walach, bugfixes, testing
```
Comment (mts299) [inline rst-authors.md:30]:
``` suggestion
- M.-T. Walach, bugfixes, testing, documentation
```

File: rst-authors.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- A.G. Burrell, documentation, bugfixes, testing
+- E.D.P. Cousins, CS10 and PSR10 convection models
+- K. Kotyk, FITACF_v3
+- P.V. Ponomarenko, DAWG Chair (2013-2019), FITACF_v2, FITACF_v3, data simulator library
+- A.S. Reimer, refactorization, documentation, bugfixes
+- A.J. Ribeiro, FITEX2, LMFIT, data simulator library
+- M.T. Schmidt, refactorization, documentation
+- S.G. Shepherd, AACGM_v2, MLT_v2, IGRF_v2, elevation_v2, TS18 convection model
+- K.T. Sterne, DAWG Co-Chair (2019-), code comments
```
Comment (mts299) [inline rst-authors.md:28]:
``` suggestion
- K.T. Sterne, DAWG Co-Chair (2019-), code comments, documentation
```

File: rst-authors.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- A.G. Burrell, documentation, bugfixes, testing
+- E.D.P. Cousins, CS10 and PSR10 convection models
+- K. Kotyk, FITACF_v3
+- P.V. Ponomarenko, DAWG Chair (2013-2019), FITACF_v2, FITACF_v3, data simulator library
+- A.S. Reimer, refactorization, documentation, bugfixes
+- A.J. Ribeiro, FITEX2, LMFIT, data simulator library
+- M.T. Schmidt, refactorization, documentation
```
Comment (mts299) [inline rst-authors.md:26]:
```
- M.T. Schmidt, refactorization, documentation, enhancements, bugfixes
```

**Comment by ecbland** [comment]
> Where should we host this? Sorry, maybe this was mentioned in our meeting. I can add it to readthedocs if you want.

For now it's going into the top level directory of RST, which is what we discussed at an earlier meeting. But I don't feel strongly about moving it somewhere else. So long as it is visible to users (readthedocs would satisfy that)

File: rst-authors.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+- C. Hanuise
+- N. Mattin
+- K. Oksavik
+- J.M. Ruohoniemi
+- J.P. Villain
+- S. Wing
+
+The RST is now maintained collaboratively by the SuperDARN Data Analysis Working Group (DAWG). Members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. These contributions are credited to:
+
+- D.D. Billett, documentation, testing
```
Comment (mts299) [inline rst-authors.md:18]:
``` suggestion
- D.D. Billett, documentation, testing, enhancements
```

**Review by mts299 — COMMENTED** [review]
In recent updates, I have noticed people doing 

I added enhancements like updating/improving things in RST. I hope that makes sense.

**Comment by mts299** [comment]
> For now it's going into the top level directory of RST, which is what we discussed at an earlier meeting. But I don't feel strongly about moving it somewhere else. So long as it is visible to users (readthedocs would satisfy that)

Sounds good! My thought was it won't appear like the README, so it is a bit tucked away but we can discuss that later as well.

**Review by mts299 — APPROVED** [review]

**Pull request merged**